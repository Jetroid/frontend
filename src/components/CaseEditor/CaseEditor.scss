@import "../../UniversalStyles";
.addCaseForm {
  padding: 3em;
}

.addCaseForm h4 {
  font-weight: 400;
  padding-top: 1em;
}

.choice {
  padding-bottom: 0.5em;
}

.choicelist {
  opacity: 1;
}

.itemPics {
  .col {
    height: 11.3rem;
    @include media-breakpoint-down(sm) {
      height: 15.2rem;
    }
    @include media-breakpoint-only(xs) {
      height: 17.2rem;
    }
    .box {
      position: relative;
      overflow: hidden;
      height: 160px;
      .checkbox {
        position: absolute;
        width: 1.4rem;
        height: 1.4rem;
        right: 0;
        background: #fff;
      }
      small {
        background: #fff;
        position: absolute;
        left: 0;
        width: 100%;
        bottom: 0;
      }
      text-align: center;
      &.lead {
        border: 4px solid $interactive-color;
        .checkbox {
          background-image: url("../../img/check.png");
          background-repeat: no-repeat;
          background-size: 1.4rem;
        }
      }
    }
  }
}
