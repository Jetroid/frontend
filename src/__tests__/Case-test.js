import React from "react";
import renderer from "react-test-renderer";

import { Case } from "../containers/Case";
import CaseDetails from "../components/CaseDetails";
import CaseEditor from "../components/CaseEditor";
import ItemDetails from "../components/ItemDetails/ItemDetails";
import { IntlProvider } from "react-intl";
import { getBestMatchingMessages } from "../utils/l10n";

import { MemoryRouter } from "react-router";
import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
import getMuiTheme from "material-ui/styles/getMuiTheme";
const muiTheme = getMuiTheme({});
import { mountWithIntl } from "../helpers/intl-enzyme-test-helper.js";
import caseData from "./case_data.json";
import intlProps from "../helpers/intl-props-test-helper.js";
import afterPromises from "../helpers/afterPromises";
import injectTapEventPlugin from "react-tap-event-plugin";
injectTapEventPlugin();

let fetchMock = require("fetch-mock");

fetchMock.get(
  process.env.REACT_APP_API_URL + "/case/123",
  JSON.stringify({ data: caseData })
);
fetchMock.get(
  process.env.REACT_APP_ASSETS_URL + "countries/fullname/Italy.svg",
  JSON.stringify({ data: caseData })
);

jest.mock("../components/Gallery");
jest.mock("../components/LazyBodyEditor");
jest.mock("../utils/s3upload");
jest.mock("material-ui/FloatingActionButton");
jest.mock("../components/BookmarkToggle");
// jest.mock("material-ui/TextField");
jest.mock("material-ui/RaisedButton");
jest.mock("material-ui/RadioButton");
jest.mock("material-ui/DatePicker");
jest.mock("material-ui/SelectField");
jest.mock("material-ui/SvgIcon");
jest.mock("react-geosuggest");
jest.mock("material-ui-chip-input");

function setup() {
  const props = {
    intl: intlProps,
    auth: {
      getProfile: cb => cb({}),
      getToken: () => "foo",
      isAuthenticated: () => true
    },
    location: { pathname: "/method/123" },
    match: { params: { nodeID: 123 } },
    toggleFeatured: function() {},
    data: caseData
  };

  const enzymeWrapper = mountWithIntl(
    <MemoryRouter>
      <MuiThemeProvider muiTheme={muiTheme}>
        <Case {...props} />
      </MuiThemeProvider>
    </MemoryRouter>
  );
  return {
    props,
    enzymeWrapper
  };
}

describe("containers", () => {
  describe("Case", () => {
    it("should render proper data for case", done => {
      const { enzymeWrapper } = setup();
      afterPromises(done, () => {
        expect(enzymeWrapper.find(".sub-heading").length).toBe(1);
        expect(enzymeWrapper.find("h2.case-title").text()).not.toBe("");
      });
    });
  });
});

let props = setup().props;
let locale = "en-US";
props["details"] = CaseDetails;
let messages = getBestMatchingMessages(locale);

test("ItemDetails for Case renders correctly", () => {
  const tree = renderer
    .create(
      <IntlProvider locale={locale} messages={messages}>
        <MemoryRouter><ItemDetails {...props} /></MemoryRouter>
      </IntlProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test("Case editor renders correctly", () => {
  const tree = renderer
    .create(
      <IntlProvider locale={locale} messages={messages}>
        <MemoryRouter>
          <MuiThemeProvider muiTheme={muiTheme}>
            <CaseEditor {...props} type="case" thing={caseData} />
          </MuiThemeProvider>
        </MemoryRouter>
      </IntlProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
