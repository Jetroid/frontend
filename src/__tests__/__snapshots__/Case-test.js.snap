// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Case editor renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="main-contents"
  >
    <div
      className="detailed-case-component container-fluid"
    >
      <div
        className="hidden-sm-down sidepanel hidden-sm-down col col-md-3"
      />
      <div
        className="main-area col-12 col-md-8"
      >
        <div
          className="case-box"
        >
          <div
            className="sub-heading title-edit"
          >
            <label
              htmlFor="title"
            />
          </div>
          <Textfield
            className="custom-field"
            disabled={false}
            errorText={undefined}
            floatingLabelText=""
            fullWidth={true}
            hintText={null}
            name="title"
            onBlur={[Function]}
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder={undefined}
            type="text"
            value="Agenda 21 Locale (Province of Forli-Cesena, Italy)"
          />
          <div>
            <p
              className="sub-heading"
            />
          </div>
          <div>
            <div>
              <p
                className="sub-heading"
              />
            </div>
          </div>
          <div
            className="case-location"
          >
            <div>
              <p
                className="sub-heading"
              />
              <div
                className="location"
              />
            </div>
            <p
              className="sub-heading"
            />
            <div>
              <div
                className="date-field start_date"
              />
            </div>
            <div>
              <div
                className="date-field end_date"
              />
            </div>
            <p
              className="sub-heading"
            />
            <div>
              <div
                className="list links"
              />
            </div>
          </div>
          <p
            className="sub-heading"
          />
          <div
            className="itemPics pb-1 row"
          >
            <div
              className="pb-1 col col-sm-6 col-md-3"
            >
              <div
                className="box lead"
              >
                <div
                  className="checkbox"
                  onClick={[Function]}
                />
                <div
                  className="trash"
                  onClick={[Function]}
                />
                <img
                  alt=""
                  className="img-fluid"
                  src="http://uploads.participedia.xyz.s3-website-us-east-1.amazonaws.com/agenda21%2520.jpg"
                  style={
                    Object {
                      "height": "100%",
                      "objectFit": "contain",
                      "width": "100%",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="p-0 col col-md-12"
            >
              <div>
                Sorry, upload only works if you're logged in.
              </div>
            </div>
          </div>
          <div>
            <div
              className="list videos"
            />
          </div>
          <p
            className="sub-heading"
          />
          <div
            className="tags-field"
          />
        </div>
        <div>
          <div>
            <div>
              <label
                className="sub-heading"
                htmlFor="body_en"
              />
            </div>
            <div
              className="related-content"
            >
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
                <div
                  className="facilitated"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
                <div
                  className="number_of_meeting_days"
                >
                  <Textfield
                    fullWidth={true}
                    name="number_of_meeting_days"
                    onChange={[Function]}
                    value="4"
                  />
                </div>
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
                <div
                  className="ongoing"
                />
              </div>
              <div>
                <p
                  className="sub-heading"
                />
                <div
                  className="staff_type"
                >
                  <Textfield
                    fullWidth={true}
                    name="staff_type"
                    onChange={[Function]}
                    value=""
                  />
                </div>
              </div>
              <div>
                <p
                  className="sub-heading"
                />
                <div
                  className="who_else_supported_the_initiative"
                >
                  <Textfield
                    fullWidth={true}
                    name="who_else_supported_the_initiative"
                    onChange={[Function]}
                    value=""
                  />
                </div>
              </div>
              <div
                className="pb-1"
              >
                <p
                  className="sub-heading"
                />
              </div>
              <div
                className="pb-1"
              >
                <p
                  className="sub-heading"
                />
              </div>
              <div
                className="pb-1"
              >
                <p
                  className="sub-heading"
                />
              </div>
               
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</form>
`;

exports[`ItemDetails for Case renders correctly 1`] = `
<div>
  <div
    className="main-contents"
  >
    <div
      className="detailed-case-component container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="hidden-sm-down sidepanel hidden-sm-down col col-md-3"
        >
          <div />
          <div />
          <div>
            <div
              className="case-map"
            >
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "",
                  }
                }
              />
              <p
                className="case-location"
              >
                Forl√¨
                , 
                Italy
              </p>
            </div>
            <p
              className="sub-heading"
            />
            <div />
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="issue"
              >
                <a
                  className="tag"
                  href="/search?issue=Budgeting"
                  onClick={[Function]}
                >
                  Budgeting
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="specific_topic"
              >
                <a
                  className="tag"
                  href="/search?specific_topic=Sustainable%20Development"
                  onClick={[Function]}
                >
                  Sustainable Development
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="links"
              >
                <div
                  className="link"
                >
                  link 1
                </div>
                <div
                  className="link"
                >
                  link 2
                </div>
                <div
                  className="link"
                >
                  link 3
                </div>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="communication_mode"
              >
                <a
                  className="tag"
                  href="/search?communication_mode=Discussion,%20Dialogue,%20or%20Deliberation"
                  onClick={[Function]}
                >
                  Discussion, Dialogue, or Deliberation
                </a>
              </div>
            </div>
            <div />
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="decision_method"
              >
                <a
                  className="tag"
                  href="/search?decision_method=Getting%20sense%20of%20room/group%20(i.e.%20no%20formal%20voting)"
                  onClick={[Function]}
                >
                  Getting sense of room/group (i.e. no formal voting)
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="facetoface_online_or_both"
              >
                <a
                  className="tag"
                  href="/search?facetoface_online_or_both=facetoface"
                  onClick={[Function]}
                >
                  facetoface
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="facilitated"
              >
                <a
                  className="tag"
                  href="/search?facilitated=true"
                  onClick={[Function]}
                />
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="voting"
              >
                <a
                  className="tag"
                  href="/search?voting=true"
                  onClick={[Function]}
                >
                  true
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="number_of_meeting_days"
              >
                <a
                  className="tag"
                  href="/search?number_of_meeting_days=4"
                  onClick={[Function]}
                >
                  4
                </a>
              </div>
            </div>
            <div />
            <div />
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="targeted_participants_public_role"
              >
                <a
                  className="tag"
                  href="/search?targeted_participants_public_role=Stakeholder%20Organizations"
                  onClick={[Function]}
                >
                  Stakeholder Organizations
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="targeted_audience"
              >
                <a
                  className="tag"
                  href="/search?targeted_audience=General%20Public"
                  onClick={[Function]}
                >
                  General Public
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="participant_selection"
              >
                <a
                  className="tag"
                  href="/search?participant_selection=Open%20to%20all%20with%20targeted%20recruitment"
                  onClick={[Function]}
                >
                  Open to all with targeted recruitment
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="type_of_funding_entity"
              >
                <a
                  className="tag"
                  href="/search?type_of_funding_entity=Regional%20Government%20(e.g.%20State,%20Provincial,%20Territorial)"
                  onClick={[Function]}
                >
                  Regional Government (e.g. State, Provincial, Territorial)
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="typical_implementing_entity"
              >
                <a
                  className="tag"
                  href="/search?typical_implementing_entity=Regional%20Government%20(e.g.%20State,%20Provincial,%20Territorial)"
                  onClick={[Function]}
                >
                  Regional Government (e.g. State, Provincial, Territorial)
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="typical_sponsoring_entity"
              >
                <a
                  className="tag"
                  href="/search?typical_sponsoring_entity=International%20Organization"
                  onClick={[Function]}
                >
                  International Organization
                </a>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="ongoing"
              />
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="start_date"
              >
                <span>
                  5/14/2003
                </span>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="end_date"
              >
                <span>
                  2/11/2011
                </span>
              </div>
            </div>
            <div>
              <p
                className="sub-sub-heading"
              />
              <div
                className="updated_date"
              >
                <span>
                  5/27/2016
                </span>
              </div>
            </div>
            <div />
            <div />
            <div />
          </div>
          <div />
        </div>
        <div
          className="main-area col-12 col-md-8"
        >
          <div
            className="case-box"
          >
            <h2
              className="category"
            />
            <h2
              className="case-title"
            >
              Agenda 21 Locale (Province of Forli-Cesena, Italy)
            </h2>
            <ul
              className="icons-mobile clearfix hidden-sm-up"
            >
              <li />
              <li>
                <div
                  className="SocialMediaShareButton SocialMediaShareButton--facebook"
                  onClick={[Function]}
                  role="button"
                  style={Object {}}
                  tabIndex="0"
                >
                  <img
                    alt=""
                    src="pp-case-icon-fb.svg"
                  />
                </div>
              </li>
              <li>
                <div
                  className="SocialMediaShareButton SocialMediaShareButton--twitter"
                  onClick={[Function]}
                  role="button"
                  style={Object {}}
                  tabIndex="0"
                >
                  <img
                    alt=""
                    src="pp-case-icon-tw.svg"
                  />
                </div>
              </li>
            </ul>
            <div
              className="authorship-details"
            >
              <p
                className="author-line"
              >
                First submitted by¬†
                <a
                  href="/users/undefined"
                  onClick={[Function]}
                >
                  Paolo Galassi
                </a>
              </p>
              <p
                className="date-line"
              >
                <time
                  dateTime="2012-01-07T12:27:34.123Z"
                  title="2012-01-07T12:27:34.1234Z"
                >
                  6 years ago
                </time>
              </p>
              <p
                className="author-line"
              >
                Most recent changes by¬†
                <a
                  href="/users/undefined"
                  onClick={[Function]}
                >
                  Paolo Galassi
                </a>
              </p>
              <p
                className="date-line"
              >
                <time
                  dateTime="2016-05-27T21:46:39.123Z"
                  title="2016-05-27T21:46:39.1234Z"
                >
                  1 year ago
                </time>
              </p>
            </div>
            <div
              className="case-html"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<p><em>NOTE: The English translation of this case can be found at <a href=\\"http://participedia.net/cases/local-agenda-21-province-forli-cesena-italy\\">http://participedia.net/cases/local-agenda-21-province-forli-cesena-italy</a>. </em></p><h3>Problems and Purpose</h3><p>Agenda 21 considera l‚Äô<em>interazione</em> tra la dimensione <em>sociale</em>, <em>ambientale</em> ed <em>economica </em>come la chiave per la conservazione e la gestione delle risorse del territorio. A tal proposito gli impegni da prendere devono conformarsi alle caratteristiche dei territori a cui si riferiscono e rispondere alle necessit√† delle popolazioni che si impegnano ad adottarli: le tematiche di discussione e intervento possono essere tra le pi√π svariate, lotta alla povert√†, cambiamento dei modelli di produzione e consumo, dinamiche demografiche, conservazione e gestione delle risorse naturali, protezione dell'atmosfera, degli oceani e della biodiversit√†, prevenzione della deforestazione, promozione di un' agricoltura sostenibile, fonti di energia rinnovabili, raccolta differenziata, ecc.</p><h3>History</h3><p>Agenda 21 - ‚Äúl‚Äôelenco delle azioni da compiere nel corso del XXI secolo perch√© lo sviluppo sostenibile sia raggiunto‚Äù - √® un progetto dell‚ÄôONU nato in occasione della Conferenza Mondiale su ‚ÄúAmbiente e Sviluppo Sostenibile‚Äù tenutasi a Rio de Janeiro nel 1992, il cui obiettivo √® creare un modello concreto di sviluppo che risponda alle necessit√† del presente senza compromettere la capacit√† delle generazioni future di soddisfare le proprie esigenze. In seguito a tale Conferenza 178 governi di tutto il mondo, tra cui l'Italia, hanno sottoscritto e adottato questo documento di intenti destinato alla promozione del cos√¨ detto <em>sviluppo sostenibile,</em> che tenendo conto di variabili sociali, ambientali ed economiche si propone di individuare in un determinato contesto territoriale eventuali elementi di incompatibilit√† o conflitto esistenti tra attivit√† socio-economiche e politiche di protezione e salvaguardia dell'ambiente.</p><p>Nel capitolo 28 dell‚ÄôAgenda 21 l‚ÄôONU riconosce agli <em>Enti Locali</em> il ruolo di <em>promotori</em> nei processi che coinvolgano tutte le realt√† rappresentative della societ√† civile, rafforzandone il ruolo e la partecipazione, e che portino alla stesura di <em>Piani d‚ÄôAzione </em>per lo sviluppo sostenibile a livello locale. Le autorit√† locali - Comuni, Provincie, Comunit√† Montane, Regioni, ecc. - sono invitate a giocare un ruolo chiave nell'educare, mobilitare e rispondere al pubblico per la promozione di uno sviluppo sostenibile, attraverso la consultazione e la costruzione di consenso, in una sorta di scambio reciproco in cui √® a loro volta possibile assorbire dalle comunit√† locali e dalle imprese le informazioni necessarie per la formulazione e l‚Äôapplicazione di nuove strategie.</p><h3>Originating Entities and Funding</h3><p>In Emilia Romagna, nel caso specifico della provincia di Forl√¨ e Cesena, l‚Äôadozione dell‚ÄôAgenda 21 Locale ha interessato oltre ai due capoluoghi, il comune di Bagno di Romagna in collaborazione con i comuni di Sarsina e di Verghereto, i comuni della costa adriatica romagnola - Cesenatico, Gatteo, San Mauro Pascoli e Savignano Sul Rubicone - i comuni della Comunit√† Montana dell‚ÄôAppennino Forlivese in collaborazione con la Comunit√† Montana dell'Acquacheta. Ci√≤ che risulta evidente ad un primo sguardo √® l‚Äô<em>eterogeneit√†</em> degli attori partecipanti e la <em>variet√† </em>delle <em>tipologie </em>di <em>stakeholders</em> prendenti parte al processo partecipativo: singoli cittadini, Forze dell‚ÄôOrdine, Associazioni di Categoria, Sindacati Agricoli e non, Associazioni di Consumatori e Ambientaliste, Ordini Professionali, Universit√†, Scuole, Enti di Formazione, Istituzioni Religiose, Associazioni sportive, Banche, Fondazioni, Imprese, Istituzioni, Associazioni Studentesche, Consiglieri Provinciali, Partiti e Parlamentari.</p><h3>Participant Selection</h3><p>Tale processo partecipativo si √® strutturato principalmente in due fasi, una <em>interna</em> ed una <em>esterna</em>. La prima √® riassumibile nell‚Äôincontro avvenuto presso la Giunta Provinciale, alla presenza del Presidente e di tutti gli assessori, mirato all‚Äôelaborazione della <strong>Dichiarazione di Sostenibilit√† della Giunta</strong>, nella quale si riconosce lo sviluppo sostenibile come linea strategica delle politiche a livello provinciale: in tale documento, nel perseguire politiche per lo sviluppo ecosostenibile del territorio, l‚Äôente si impegna a <em>operare nella tutela dell‚Äôambiente naturale assicurando una gestione delle risorse efficace ed efficiente</em>, <em>indirizzare lo sviluppo economico endogeno verso forme imprenditoriali che privilegino ricerca ed innovazione</em>, <em>assicurando una crescita omogeneamente diffusa, valorizzando la qualit√† del lavoro, mantenendo e possibilmente aumentando la coesione sociale, territoriale e multietnica, cercando di assicurare l‚Äôequilibrio delle opportunit√† sociali per le fasce deboli con una garanzia di welfare</em>. In quest‚Äôottica il <em>tentativo</em> di promozione di un maggior livello di democrazia e partecipazione dei cittadini nei processi decisionali rappresenta il metodo di attivazione dei processi di sostenibilit√†.</p><p>La fase<em> esterna</em> ha invece previsto in un primo momento il coinvolgimento degli <em>stakeholders: </em>le procedure di individuazione e coinvolgimento dei portatori d‚Äôinteresse sono state guidate da uno dei principi della <em>governance </em>definiti nel Libro Bianco della Commissione Europea, il ‚Äúprincipio di apertura‚Äù, che prevede il coinvolgimento di tutti i portatori d‚Äôinteresse di una comunit√† nei processi partecipativi, ovvero in questo caso, l‚Äôindividuazione di tutti i soggetti che a livello provinciale sono portatori di un <em>interesse condiviso</em>. Mediante tale processo di coinvolgimento, sviluppato per area <em>territoriale,</em> ci si propone di incentivare la partecipazione di tutti i diversi interessi presenti nel territorio provinciale. La selezione degli invitati agli incontri tematici √® stata fatta anche coerentemente con il tema di discussione dei singoli workshop in questione, tuttavia essendo Agenda 21 Locale un processo per definizione <em>volontario</em>, sono stati accolti all‚Äôinterno del Forum anche tutti quei cittadini che spontaneamente ne hanno chiesto di far parte. In occasione di ogni incontro sono state inviate lettere invito personalizzate con il programma dei lavori, alla cui spedizione seguiva poi un <em>re-call</em> telefonico per la conferma della presenza.</p><h3>Deliberation, Decisions, and Public Interaction</h3><p>Per la fase operativa del processo √® stata utilizzata la metodologia EASW¬Æ - <em>European Awareness Scenario Workshop</em> ‚Äì della Commissione Europea, basata su due attivit√† principali: in un primo momento la ‚Äú<strong>definizione di scenari</strong>‚Äù e in seguito la ‚Äú<strong>proposta di idee</strong>‚Äù. In generale, l‚ÄôEASW serve a stimolare la partecipazione democratica nelle scelte legate al miglioramento delle condizioni di vita nelle comunit√†. Consente ai partecipanti di scambiarsi informazioni, discutere i temi ed i processi che governano lo sviluppo tecnologico e l'impatto delle tecnologie sull'ambiente naturale e sociale, stimolandone la capacit√† di identificare e pianificare soluzioni concrete ai problemi esistenti.</p><p>Nella prima fase di lavoro i partecipanti, suddivisi in gruppi di origine omogenei, hanno costruito una propria autonoma visione del futuro. Al tal fine √® stato necessario stimolare un approccio integrato che tenesse conto della complessit√† e delle interdipendenze che caratterizzano i contributi forniti. Indipendentemente dal tema in discussione, ai partecipanti si √® richiesto di definire il pi√π possibile le caratteristiche future dello scenario che si apprestavano a immaginare, in riferimento al quale sono stati chiamati a suggerire possibili ed efficaci modi concreti di realizzazione. Ciascuno scenario ha messo in evidenza ci√≤ che il gruppo si auspicava che accadesse relativamente alla sostenibilit√† del proprio territorio in un margine di previsione di 10 anni, specificando i fattori chiave di successo.</p><p>Nella seconda fase i partecipanti, suddivisi in gruppi misti, hanno formulato proposte operative sulle azioni necessarie da intraprendere per realizzare lo scenario futuro costruito. In questa attivit√† ai partecipanti √® stato chiesto di essere concisi e concreti, di spiegare i passaggi da realizzare e di individuare i soggetti dell‚Äôazione (CHI) e le modalit√† di realizzazione (COME). In ogni incontro tematico i partecipanti sono stati invitati ad individuare obiettivi strategici, specifici, azioni, attori e priorit√†.</p><h3>Influence, Outcomes, and Effects</h3><p>Il <strong>14 maggio 2003</strong> √® avvenuta la <strong>Costituzione</strong> e l‚Äô<strong>Attivazione del Forum Provinciale</strong> <strong>Permanente</strong> di Agenda 21 Locale e la scelta delle aree tematiche di approfondimento dei <em>workshop tematici</em>: all‚Äôincontro erano presenti pi√π di un centinaio di portatori d‚Äôinteresse del mondo istituzionale, economico, produttivo, agricolo, dell‚Äôassociazionismo culturale, ambientalista e del mondo del volontariato. Grazie alla collaborazione con un <em>team</em> di ricerca dell‚ÄôUniversit√† di Siena √® stata presentata un‚Äôanalisi dell‚ÄôImpronta Ecologica della Provincia, ovvero l‚Äôanalisi dei consumi e degli impatti ambientali di un territorio in rapporto alla disponibilit√† di quest‚Äôultimo e un‚Äôanalisi <em>emergetica</em>, che considera i processi necessari alla produzione di energia e i materiali utilizzati in un tale processo, includendo sia fattori economici - l‚Äôenergia e la materia utilizzate quantificabili anche in termini monetari - sia fattori ambientali, legati al consumo di risorse espresso in unit√† di energia solare: entrambe le metodologie hanno evidenziato il forte impatto delle attivit√† antropiche sul territorio provinciale.</p><p>Il Forum si √® composto di tre workshop, svoltisi in diversi luoghi della provincia e in giornate diverse, nei quali la presenza dei partecipanti √® stata affiancata da tutor tecnici con l‚Äôobiettivo di arricchire e approfondire da un punto di vista pi√π strettamente tecnico il lavoro. Le tematiche di riferimento sono emerse dalla riflessione circa gli ambiti ‚Äúsensibili‚Äù suscettibili di approfondimento partecipato, considerando le analisi e gli approfondimenti sull‚Äôimpronta ecologica del territorio.</p><p>Il primo workshop -19 settembre 2003-¬† ha avuto come argomento ‚ÄúI consumi e gli stili di vita‚Äù, dal quale sono emersi sottogruppi di discussione sui temi di <em>energia</em>, <em>rifiuti</em>, <em>trasporti e logistica </em>e<em>¬† strumenti di sostenibilit√†. </em></p><p>‚ÄúL‚Äôagricoltura e il territorio‚Äù √® stato l‚Äôargomento del secondo incontro, svoltosi il 17 ottobre 2003, che ha visto la formazione di sottotematiche riguardanti la <em>gestione delle risorse idriche</em>, <em>l‚Äôuso del territorio</em>, <em>le produzioni ecocompatibili, l‚Äôagricoltura del futuro</em>.</p><p>‚ÄúL‚Äôinnovazione e lo sviluppo‚Äù √® stato il titolo dell‚Äôultima parte, svoltasi il 7 novembre 2003, che ha visto discussioni attorno alle tematiche di: <em>informazione, formazione e partecipazione</em> come presupposti per la diffusione di informazione, condivisione, conoscenza e approfondimento delle problematiche ambientali; <em>ricerca e innovazione</em> come punto di partenza per la nascita di nuovi filoni d‚Äôindagine e per la conoscenza del territorio nelle sue forme pi√π peculiari; <em>nuove personalit√† e formazione, </em>riguardo i contributi e gli incentivi per il supporto alla creazione di profili professionali qualificati ricercati sul mercato del lavoro; <em>identit√† e territorio,</em> riguardante la nascita di un nuovo senso di percezione del territorio che valorizza le specificit√† e i tratti caratteristici.</p><p>L‚Äô<strong>11 febbraio 2004</strong> √® stato infine presentato il <strong>Piano d‚ÄôAzione Locale d‚ÄôArea</strong>: la fase di elaborazione del documento ha comportato la raccolta del lavoro svolto da ciascun forum tematico. Tale fase rappresenta il momento di individuazione dei nuovi obiettivi, stimoli, idee, ragionamenti intorno al tema della ‚Äúsostenibilit√†‚Äù, in modo integrato con le attivit√† che gi√† l‚Äôente svolge, con l‚Äôobiettivo di costruire un quadro il pi√π completo possibile di ci√≤ che viene fatto sul territorio e di quello che i portatori d‚Äôinteresse rilevano come <em>prioritario</em> da realizzare.</p><h3>Analysis and Criticism</h3><p>Nel caso dell‚ÄôAgenda 21 Locale della provincia di Forl√¨ e Cesena il livello di partecipazione degli stakeholder all‚Äôavvio del Forum √® stato alto, ma ci√≤ che pu√≤ destare dubbi sull‚Äôefficacia del funzionamento del processo, sull‚Äôequilibrio del processo decisionale stesso, sulla sua apertura, sull‚Äôestraneit√† a conflitti di interesse e a pressioni di tipo politico-economico sono i dati riguardanti l'<em>eterogeneit√†</em> della composizione del corpo partecipante, nonch√© la provenienza dei rispettivi attori: di questi infatti ben il 46% era costituito da rappresentanti di Istituzioni, il 17% da Imprese, l‚Äô11% da Associazioni di Categoria, il 10% da associazioni Ambientaliste, il 6%da Enti di Formazione, il 4% da sindacati agricoli, solamente il 3% da cittadini, e un 1% spettava rispettivamente a Ordini Professionali, Scuole e Gruppi di Partito e Parlamentari.</p><h3>External Links</h3><p><a href=\\"http://www.a21italy.it\\">http://www.a21italy.it</a></p><p><a href=\\"http://www.agenda21.it/\\">http://www.agenda21.it/</a></p><p><a href=\\"http://cordis.europa.eu/easw/\\">http://cordis.europa.eu/easw/</a></p><p><a href=\\"http://osservatoriopartecipazionerer.ervet.it/\\">http://osservatoriopartecipazionerer.ervet.it/</a></p><p><a href=\\"http://www.provincia.fc.it/agenda21/\\">http://www.provincia.fc.it/agenda21/</a></p><p><a href=\\"http://partecipazione.regione.emilia-romagna.it/entra-in-regione/progetti/agende-21\\">http://partecipazione.regione.emilia-romagna.it/entra-in-regione/progett...</a></p><p><a href=\\"http://www.un.org/esa/dsd/agenda21/\\">http://www.un.org/esa/dsd/agenda21/</a></p><p>¬†</p>",
                }
              }
            />
          </div>
        </div>
        <div
          className="case-tools hidden-sm-down col col-md-1"
        >
          <div
            className="top-icons"
          >
            <div
              className="SocialMediaShareButton SocialMediaShareButton--facebook"
              onClick={[Function]}
              role="button"
              style={Object {}}
              tabIndex="0"
            >
              <img
                alt=""
                src="pp-case-icon-fb.svg"
              />
            </div>
            <div
              className="SocialMediaShareButton SocialMediaShareButton--twitter"
              onClick={[Function]}
              role="button"
              style={Object {}}
              tabIndex="0"
            >
              <img
                alt=""
                src="pp-case-icon-tw.svg"
              />
            </div>
            <div
              className="SocialMediaShareButton SocialMediaShareButton--linkedin"
              onClick={[Function]}
              role="button"
              style={Object {}}
              tabIndex="0"
            >
              <img
                alt=""
                src="pp-case-icon-ln.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <a
      href="/case/undefined/edit"
      onClick={[Function]}
    />
  </div>
</div>
`;
